corehq/motech/dhis2/entities_helpers.py

```diff
@@ -59,20 +59,74 @@ def send_dhis2_entities(requests, repeater, case_trigger_infos):
     Send request to register / update tracked entities
     """
     errors = []
+    info_config_entities = _update_or_register_teis(
+        requests,
+        repeater,
+        case_trigger_infos,
+        errors,
+    )
+    _create_all_relationships(
```

Comment:
Since we're passing in mostly the same parameters for this function than the previous function, how would you feel about creating a small datastructure/object containing `requests`, `repeater` and `errors`. Then each function can update this datastructure/object. I feel like it might be easier to read?